### PR TITLE
CMDCT-4087: NAAAR Dashboard Page Setup

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Adding the verbiage for the NAAAR template card and the empty state of a NAAAR dashboard upon entering `/naaar`:

Screenshot 2024-10-30 at 11 56 50 AM

Screenshot 2024-10-30 at 11 56 58 AM

**Note:** The links in the dashboard description are placeholders. 

### Related ticket(s)

CMDCT-4087

---
### How to test

- Log in as a state user
- Verify the verbiage in the template card
- Click on "Enter NAAAR"
- Verify the verbiage on the dashboard

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/verbiage/pages/naaar/naaar-dashboard.ts [hunk 1]
```diff
@@ -4,21 +4,36 @@ export default {
     body: [
       {
         type: "text",
-        content: "",
+        content:
+          "Complete one (1) report with information for applicable managed care plans and their applicable managed care programs. MMPs are considered both Medicaid and Medicare managed care plans and are not exempt from 42 CFR 438.207. Therefore, states must submit the tool for integrated plans; however, to reduce duplication, states can complete network adequacy sections of the tool (II.A.1-II.A.5) for Medicaid-only covered services. Reporting on Program of All-Inclusive Care for the Elderly (PACE) programs/plans is not required.",
```
Comment (gmrabian) [inline services/ui-src/src/verbiage/pages/naaar/naaar-dashboard.ts:8]: need link or link placeholder for `42 CFR 438.207` text as well

**Review by gmrabian — DISMISSED** [review]
non-blocking: link/placeholder missing

File: services/ui-src/src/verbiage/pages/naaar/naaar-dashboard.ts (hunk 1 above)
Comment (karla-vm) [reply services/ui-src/src/verbiage/pages/naaar/naaar-dashboard.ts]: ✅ fixed!

**Review by gmrabian — DISMISSED** [review]

**Review by bangbay-bluetiger — APPROVED** [review]

**Review by bangbay-bluetiger — APPROVED** [review]

**Review by jessabean — APPROVED** [review]
👍🏻

**Review by gmrabian — APPROVED** [review]

**Review by gmrabian — APPROVED** [review]

**Pull request merged**